src/cx_ed25519.c

```diff
@@ -180,3 +187,168 @@ int scalarmult_ed25519(BIGNUM *Qx, BIGNUM *Qy, BIGNUM *Px, BIGNUM *Py, BIGNUM *e
 
   return scalarmult(&Q, &P, e);
 }
+
+void cx_encode_int(uint8_t *v, size_t len)
```

Comment:
More that "encode/decode", this is about endianness (converting Big Endian numbers from/to Little Endian, because Ed25519 "exported data" is Little Endian but the computations in Speculos are Big Endian). I suggest renaming these functions `be2le` and `le2be`, or at least adding comments that describe their purposes.